doc/proposals/images-subspec.md

```diff
@@ -84,12 +100,35 @@ Implementing the proposal would include:
 
 ### User Stories
 
```

Comment:
I know that the proposals in this repo are not exactly in this way. But, IHMO should be described by following the story concept. E.g `As a user, would like to ... ` See https://www.mountaingoatsoftware.com/agile/user-stories. Then, we should add acceptance criteria. See [here](https://github.com/operator-framework/operator-sdk/pull/2140/files) an example over how I think that we should do it.